multi-node-test.sh

```diff
@@ -148,114 +107,106 @@ df -h" 2>&1 | dshbak -c; then
     exit 1
 fi
 
+if ! ssh -i ci_key jenkins@"${nodes[0]}" "set -x
```

Comment:
(lint) Did you forget the 'then' for this 'if'? [SC1049]
(lint) Couldn't parse this if expression. [SC1073]